scss/_card.scss

```diff
@@ -120,8 +120,8 @@
 }
 
 .card-header-pills {
-  margin-right: -$card-cap-padding-x / 2;
-  margin-left: -$card-cap-padding-x / 2;
+  margin-right: $card-header-pills-margin-right;
+  margin-left: $card-header-pills-margin-left;
```

Comment:
Same here—`-x` and `-y`, not `-right` etc.

Reply:
@mdo isn't `y`  for `top` / `bottom` values ? 
according to naming rules what if we need to create variable for `right` and `left` separately? We can't use `x` for both...